src/cotletico_manager.py

```diff
@@ -5,24 +5,6 @@
 import match
 
 
-def schedule_from_json(filename: str):
-    data = ''
-    with open(filename) as files:
-        text = files.read()
-        data = json.loads(text)
-
-    matches = list()
-    for match_info in data:
-        match = Match(match_info["Location"], 0, match_info["Date"], match_info["Team 1"], match_info["Team 2"], [], [])
-        matches.append(match)
-
-    schedule = Schedule(matches)
-    return schedule
-
-
-schedule = schedule_from_json('../data/schedule.json')
-
-
 def teams_from_json(filename: str):
```

Comment:
Move to _team.py_ to _team_ class and rename method to **_from_json_file_**.

Reply:
Added a method from the match_generator branch